src/main/common/router/routablePath.ts

```diff
@@ -23,4 +25,28 @@ export class RoutablePath {
     const viewPath: string = split.slice(2).join('/')
     return `${featureName}/views/${viewPath}`
   }
+
+  evaluateUri (substitutions: { [key: string]: string }): string {
```

Comment:
missing tests for this? (we really need to pull this sort of stuff out to a common module)

Reply:
written tests